docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -56,6 +44,34 @@ info:
     X-Hypothesis-Media-Type: application/vnd.hypothesis.v1+json
```

Comment:
As you noted on the call, the `X-` prefix needs to be removed to match the implementation.